libafl_qemu/src/hooks.rs

```diff
@@ -363,36 +362,6 @@ where
     }
 }
 
-impl<I, QT> QemuHooks<QT, NopState<I>>
```

Comment:
What does that have to do with anything?

Reply:
It was unused and some changes I made (namely, I moved NopState into a test module) had invalidated it.